### DISCUSSION TITLE
Cache Revalidate

### DISCUSSION THREAD

**Discussion by Liion91** [arthurfiorette/axios-cache-interceptor#631] [Q&A] [answered]
Hi, 
there is an option for caching only for a certain time? 
For example I want cache one response for one day and then revalidate the response, can I make this?
Thanks.

**Comment by arthurfiorette** [comment, marked as answer]
Probably you want to use TTL.
https://axios-cache-interceptor.js.org/config/request-specifics#cache-ttl
